pipeline/subgraph.go

```
@@ -143,20 +143,38 @@ func (h *SubgraphHandler) Execute(ctx context.Context, node *Node, pctx *Pipelin
 	// with the correct OutcomeBudgetExceeded status. Mapping it to fail
 	// here would trip the engine's strict-failure-edges rule before the
 	// parent's budget check runs, masking the real reason for the halt.
+	//
+	// OutcomeValidationOverridden gets the same Success treatment for
+	// parent routing — the child's override doesn't redirect the parent's
+	// edge selection (parent decides its own routing). The override flag
+	// is propagated up via ChildOverride below; the parent's engine
+	// absorbs it into the sticky list and the terminal-status rule
+	// flips Success → ValidationOverridden at the parent level.
 	status := OutcomeSuccess
```

golangci-lint (2.12.2)

[error] ineffectual assignment to status

(ineffassign)